pkg/cmd/release/create/create.go

```diff
@@ -99,20 +106,25 @@ func NewCmdCreate(f *cmdutil.Factory, runF func(*CreateOptions) error) *cobra.Co
 			Create a release and start a discussion
 			$ gh release create v1.2.3 --discussion-category "General"
 		`),
-		Args: cmdutil.MinimumArgs(1, "could not create: no tag name provided"),
+		Args: cmdutil.MinimumArgs(0, "something unexpected happened, please report this error"),
 		RunE: func(cmd *cobra.Command, args []string) error {
 			if cmd.Flags().Changed("discussion-category") && opts.Draft {
-				return errors.New("Discussions for draft releases not supported")
+				return errors.New("discussions for draft releases not supported")
 			}
 
 			// support `-R, --repo` override
 			opts.BaseRepo = f.BaseRepo
 			opts.RepoOverride, _ = cmd.Flags().GetString("repo")
 
-			opts.TagName = args[0]
+			opts.TagName = ""
+			assetsIndex := 0
+			if len(args) > 0 {
+				opts.TagName = args[0]
+				assetsIndex = 1
+			}
```

Comment:
```suggestion
			if len(args) > 0 {
				opts.TagName = args[0]
				opts.Assets, err = shared.AssetsFromArgs(args[1:])
				if err != nil {
 				    return err
 			        }
			}
```
This looks like it could be simplified a bit.